proxy/http/HttpBodyFactory.cc

```diff
@@ -372,8 +372,8 @@ HttpBodyFactory::fabricate(StrList *acpt_language_list, StrList *acpt_charset_li
   char *buffer;
   const char *pType = context->txn_conf->body_factory_template_base;
   const char *set;
-  HttpBodyTemplate *t = nullptr;
-  HttpBodySet *body_set;
+  HttpBodyTemplate *t   = nullptr;
```

Comment:
Seems better to also move this to right before line 412 to make code more readable.